trace/redis/redis.go

```diff
@@ -0,0 +1,79 @@
+package redis
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/beatlabs/patron/trace"
+	"github.com/go-redis/redis/v7"
+	"github.com/opentracing/opentracing-go"
+	"github.com/opentracing/opentracing-go/ext"
+)
+
+const (
+	// RedisComponent definition
+	RedisComponent = "redis"
+	// RedisDBType description
+	RedisDBType = "In-memory"
+)
+
+// Span starts a new Redis child span with specified tags.
+func Span(ctx context.Context, opName, cmp, dbType, instance, stmt string,
```

Comment:
This should not be used outside of this package so it should be unexported. correct?
```suggestion
func span(ctx context.Context, opName, cmp, dbType, instance, stmt string,
```